testing/go/smoke_test.go

```diff
@@ -458,6 +458,92 @@ func TestSmokeTests(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "BETWEEN",
```

Comment:
Don't we have some skipped tests waiting on this?

Reply:
We might, I'll check

Reply:
Doesn't look like it!